### PR TITLE
pPb relvals updates

### PR DESCRIPTION
**Pull request by fabozzi**
New pPb data relval
Update pPb MC relval to use HLT:PIon

### CODE REVIEW

**Comment by fabozzi** [comment]
@mandrenguyen 
you are interested to this

**Comment by fabozzi** [comment]
please test

File: Configuration/PyReleaseValidation/python/relval_standard.py [hunk 1]
```diff
@@ -105,6 +105,7 @@
 workflows[140.51] = ['',['RunHI2010','REPACKHID','RECOHID11St3','HARVESTDHI']]
 workflows[140.52] = ['',['RunHI2010','RECOHID10','RECOHIR10D11','HARVESTDHI']]
 workflows[140.53] = ['',['RunHI2011','RECOHID11','HARVESTDHI']]
+workflows[140.54] = ['',['RunPA2013','RECOUP15_PPbData','HARVESTUP15_PPbData']]
```
Comment (slava77) [inline Configuration/PyReleaseValidation/python/relval_standard.py]: The data input is 2013 (Run1) while the era setup is RECOUP15 (which points to Run2 era customizations).
This is asking for trouble and will probably crash in CSC.

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 2]
```diff
@@ -1533,6 +1536,8 @@ def lhegensim(fragment,howMuch):
 # for Run2 PPb workflows
 steps['HARVESTUP15_PPb']=merge([{'--conditions':'auto:run2_mc_pa','--era':'Run2_2016_pA'}, steps['HARVESTMINUP15']])
 
+steps['HARVESTUP15_PPbData']=merge([{'--conditions':'auto:run2_data','-s':'HARVESTING:dqmHarvesting','--scenario':'pp' }, steps['HARVESTDHI']])
```
Comment (slava77) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: for completeness, better use the same Run2_2016_pA era

**Comment by slava77** [comment]
It looks like we need to add Configuration/Eras/python/Era_Run1_pA_cff.py for the 2013 wf setup
similar to Configuration/Eras/python/Era_Run2_2016_pA_cff.py
and make it known to the Configuration/StandardSequences/python/Eras.py

**Comment by mandrenguyen** [comment]
@fabozzi Note that #16394 has now been updated with an era "Run1_pA", which can be used for this relval workflow.

**Comment by fabozzi** [comment]
... will re-start from today IB

**Pull request closed without merging**